### PR TITLE
NATIVE-32: Fix Webview height

### PR DESCRIPTION
**Pull request by NKlug**
Fixes the height of the webview (also upgrades the webview version).

### CODE REVIEW

File: src/modules/common/renderHtml.js [hunk 1]
```diff
@@ -32,18 +33,91 @@ const renderJS = (files: FilesStateType) => `
...
   }
+})();
+
+(function() {
+  var container = document.getElementById('measure-container')
+  function adjustHeight() {
+    if (window.postMessage.length !== 1){
+      setTimeout(adjustHeight, 200);
+    }
+    else {
```
Comment (maxammann) [inline src/modules/common/renderHtml.js]: Formatting is a bit off here

File: src/modules/common/renderHtml.js [hunk 2]
```diff
@@ -32,18 +33,91 @@ const renderJS = (files: FilesStateType) => `
...
+  <meta name="viewport" content="initial-scale=1.0, maximum-scale=1.0">
+  <style>
+    html, body {
+        margin: 0;
+        padding: 0;
+        
+        /*font-family: \${theme.fonts.contentFontFamily};*/   
+        font-size: ${theme.fonts.contentFontSize};
+        line-height: ${theme.fonts.contentLineHeight};
+        /*\${props => props.centered && css\`
```
Comment (maxammann) [inline src/modules/common/renderHtml.js]: Can probably be removed.

File: src/modules/common/renderHtml.js [hunk 3]
```diff
@@ -32,18 +33,91 @@ const renderJS = (files: FilesStateType) => `
       item.src = '${URL_PREFIX}' + newSrc
     }
   }
+})();
+
+(function() {
+  var container = document.getElementById('measure-container')
```
Comment (maxammann) [inline src/modules/common/renderHtml.js]: Can we use the body here?

File: src/modules/common/renderHtml.js (hunk 2 above)
Comment (Schedulaar) [reply src/modules/common/renderHtml.js]: no probably not; we'll need that once we use webview for categorylists as well

File: src/modules/common/renderHtml.js (hunk 3 above)
Comment (Schedulaar) [reply src/modules/common/renderHtml.js]: no, for some reason, body seems to behave indeterminately (at least for me)

File: src/modules/common/renderHtml.js (hunk 1 above)
Comment (NKlug) [reply src/modules/common/renderHtml.js]: Done.

**Pull request merged**